x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_essentials_license/rule_management/coverage_overview.ts

```diff
@@ -0,0 +1,585 @@
+              'T002.002': [expectedRule2.id],
+            },
+            unmapped_rule_ids: [],
+            rules_data: {
+              [expectedRule1.id]: {
+                activity: 'disabled',
+                name: 'Disabled rule',
+              },
+              [expectedRule2.id]: {
+                activity: 'enabled',
+                name: 'Enabled rule',
+              },
+            },
+          });
+        });
+      });
+
+      describe('source', () => {
+        it('returns response filtered by custom rules', async () => {
+          await createPrebuiltRuleAssetSavedObjects(es, [
+            createRuleAssetSavedObject({
+              rule_id: 'prebuilt-rule-1',
+              threat: generateThreatArray(1),
+            }),
+          ]);
+          await installPrebuiltRulesAndTimelines(es, supertest);
+
+          const expectedRule = await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({ rule_id: 'rule-1', threat: generateThreatArray(2) })
+          );
+
+          const body = await getCoverageOverview(supertest, {
+            source: [CoverageOverviewRuleSource.Custom],
+          });
+
+          expect(body).to.eql({
+            coverage: {
+              T002: [expectedRule.id],
+              TA002: [expectedRule.id],
+              'T002.002': [expectedRule.id],
+            },
+            unmapped_rule_ids: [],
+            rules_data: {
+              [expectedRule.id]: {
+                activity: 'disabled',
+                name: 'Custom query rule',
+              },
+            },
+          });
+        });
+
+        it('returns response filtered by prebuilt rules', async () => {
+          await createPrebuiltRuleAssetSavedObjects(es, [
+            createRuleAssetSavedObject({
+              rule_id: 'prebuilt-rule-1',
+              threat: generateThreatArray(1),
+            }),
+          ]);
+          const {
+            results: { created },
+          } = await installPrebuiltRules(es, supertest);
+          const expectedRule = created[0];
+
+          await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({ rule_id: 'rule-1', threat: generateThreatArray(2) })
+          );
+
+          const body = await getCoverageOverview(supertest, {
+            source: [CoverageOverviewRuleSource.Prebuilt],
+          });
+
+          expect(body).to.eql({
+            coverage: {
+              T001: [expectedRule.id],
+              TA001: [expectedRule.id],
+              'T001.001': [expectedRule.id],
+            },
+            unmapped_rule_ids: [],
+            rules_data: {
+              [expectedRule.id]: {
+                activity: 'disabled',
+                name: 'Query with a rule id',
+              },
+            },
+          });
+        });
+
+        it('returns all rules if both custom and prebuilt filters are specified in the request', async () => {
```

Comment:
Let's also add a test `returns all rules if neither of custom and prebuilt filters are specified in the request`